backend/BattleShipAPI/BattleShipAPI/Hubs/GameHub.cs

```diff
@@ -20,11 +20,24 @@ public class GameHub : Hub
     {
         private readonly InMemoryDB _db;
         private readonly INotificationService _notificationService;
+        private readonly Dictionary<string, IPlayerCommand> _commands = new();
 
         public GameHub(INotificationService notificationService)
         {
             _db = InMemoryDB.Instance;
             _notificationService = notificationService;
+            InitializeCommands();
+        }
+
+        private void InitializeCommands()
+        {
+            _commands["ready"] = new ReadyCommand();
+            _commands["start"] = new StartGameCommand();
+            _commands["gamemode"] = new ConfirmGameModeCommand();
+            _commands["generateboard"] = new GenerateBoardCommand();
+            _commands["addship"] = new AddShipCommand();
+            _commands["undoplacement"] = new UndoPlacementCommand();
+            _commands["redoplacement"] = new RedoPlacementCommand();
```

Comment:
Can't we move this to DB layer and default initilization? Or is it required by design patern?